### PR TITLE
Add NeuVector chart version 2.2.4

### PR DESCRIPTION
**Pull request by selvamt94**

### CODE REVIEW

**Review by cmurphy — APPROVED** [review]

File: packages/neuvector/generated-changes/overlay/questions.yaml [hunk 1]
```diff
@@ -1,11 +1,5 @@
 questions:
 #image configurations
-- variable: registry
-  default: "docker.io"
-  description: image registry
-  type: string
-  label: Image Registry
-  group: "Container Images"
```
Comment (cbron) [inline packages/neuvector/generated-changes/overlay/questions.yaml:8]: Sort of surprised this got backported into 2.6. I know we made this switch in 2.7, just want to make sure we are doing this purposefully in 2.6.

**Review by cbron — APPROVED** [review]
one comment, otherwise lgtm

File: packages/neuvector/generated-changes/overlay/questions.yaml (hunk 1 above)
Comment (selvamt94) [reply packages/neuvector/generated-changes/overlay/questions.yaml:8]: @cbron In 2.6 also, NeuVector is not using registry from questions.yaml. It is a bug, so removed from questions.yaml.  Registry need to be set only by systemDefaultRegistry. 
https://github.com/selvamt94/charts/blob/dev-v2.6/charts/neuvector/100.0.2%2Bup2.2.3/templates/controller-deployment.yaml#L59

Comment (cbron) [reply packages/neuvector/generated-changes/overlay/questions.yaml:8]: Ok makes sense

**Comment by cbron** [comment]
Needs rebase, then we can merge

**Pull request merged**